yaw_unit: "rad"

initial_pose:
  x: 0.0
  y: 0.0
  yaw: 0.0

storage_pose:
  x: 2.0
  y: 1.0
  yaw: 1.57

patient_pose:
  x: -1.0
  y: 3.0
  yaw: 3.14

emergency_pose:
  x: 0.5
  y: -2.5
  yaw: -1.57

through_poses:
  - x: 1.0
    y: 0.5
    yaw: 0.0
  - x: 1.5
    y: 0.8
    yaw: 0.3
